### DISCUSSION TITLE
Problema archiviazione

### DISCUSSION THREAD

**Discussion by Lucaccc** [iisgiua/giuaschool#548] [Richieste di aiuto]
Buongiorno,
Sto archiviando i documenti e, arrivato all'archiviazione delle circolari, mi restituisce errore 500.
A cosa può essere dovuto?
(p.s. Su Altervista)

**Comment by trinko** [comment]
E' molto probabile che la procedura sia stata interrotta dal server.
La generazione dei documenti può richiedere parecchio tempo e necessita di molte risorse del server: questo può comportare che il servizio di hosting blocchi improvvisamente la procedura mostrando un messaggio di errore, o anche una pagina completamente bianca. 

Prova con l’archiviazione parziale, utilizzando la modalità “Dall’elemento selezionato in poi” o, nel caso peggiore, selezionando solo un elemento alla volta.

NB: l'archiviazione “Dall’elemento selezionato in poi” la vedi solo se installi gli aggiornamenti successivi al rilascio della versione 1.5.4

**Reply by Lucaccc** [reply]
Si, stavo già procedendo con l'archiviazione parziale, ed è da li che ha iniziato a darmi questo problema..
